### ISSUE TITLE
perftune.py --tune disks fails on i3.4xlarge

### ISSUE THREAD

**Issue by tgrabiec** [scylladb/seastar#493]
```
sudo /usr/lib/scylla/perftune.py --tune disks --dir /var/lib/scylla
...
Writing 'noop' to /sys/devices/pci0000:00/0000:00:1d.0/nvme/nvme0/nvme0n1/queue/scheduler
Failed to write into /sys/devices/pci0000:00/0000:00:1d.0/nvme/nvme0/nvme0n1/queue/scheduler: (<class 'OSError'>, OSError(22, 'Invalid argument'), <traceback object at 0x7f42fe1e06c8>)
Writing '2' to /sys/devices/pci0000:00/0000:00:1d.0/nvme/nvme0/nvme0n1/queue/nomerges
Writing 'noop' to /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme1/nvme1n1/queue/scheduler
Failed to write into /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme1/nvme1n1/queue/scheduler: (<class 'OSError'>, OSError(22, 'Invalid argument'), <traceback object at 0x7f42fe1e8548>)
Writing '2' to /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme1/nvme1n1/queue/nomerges
```

`none` is recognized instead of `noop`:
```
$ cat /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme1/nvme1n1/queue/scheduler
[none] 
$ echo "none" | sudo tee /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme1/nvme1n1/queue/scheduler
none
```

**Comment by avikivity** [comment]
Looks like `noop` is recognized on some old kernels, and `none` on new ones:

```
$ cat /sys/block/sda/queue/scheduler 
noop [deadline] cfq 
```

```
$ cat /sys/block/nvme0n1/queue/scheduler 
[none] mq-deadline 
```

So a fix will have to see which one is supported by the kernel.

**Comment by vladzcloudius** [comment]
@avikivity @tgrabiec I have the fix to this in my queue. It had a very low priority since it was mainly used by me. But since now it seems to change soon I'll send a fix ASAP.

**Comment by tgrabiec** [comment]
Fixed in 87ab62f7d2370234c2ed8ffc8073e3fe1281a1cf
